src/contexts/Photos/sync.ts

```diff
@@ -23,8 +26,24 @@ export const startSync = async (handlers: SyncHandlers) => {
   }
 
   const syncManager = photos.sync;
+  let pendingPhotosInterval: number | null = null;
+  syncManager.onPhotosItemUploadStart(handlers.onPhotosItemUploadStart);
+  syncManager.onPhotosItemUploadProgress(handlers.onPhotosItemUploadProgress);
 
+  const startPendingPhotosUpdatePolling = () => {
```

Comment:
With this, we provide initial feedback quickly, every 500ms, after the first Photos item is uploaded, this interval is deactivated, and the pending items are updated each time a Photos item is uploaded